culinary-canvas/capstone-project/frontend/my_capstone_project/src/HomePage.jsx

```diff
@@ -21,6 +22,7 @@ function HomePage() {
     "cookies",
     "cheese",
     "salad",
+    "pasta",
```

Comment:
Question - can you help me explain what this `const queries` does? Why do we have a hardcoded list of foods? What if a user wants to have `pizza` which is not in that list?

Reply:
The API doesn't allow me to use the keyword random so i just hardcoded a list of foods when I started fetching the recipes from the API